### PR TITLE
Removing AWS creds

### PR DESCRIPTION
**Pull request by rogeruiz**
Sacrificing :lock_with_ink_pen: to the :volcano:

Pipeline still needs to fly.

6e3a153d-e8d3-4914-8236-d79205204dfa_text_hi

### CODE REVIEW

**Comment by rogeruiz** [comment]
I'm confident that it'll fly fine, but didn't want to do it on a monday night 🦉

**Comment by pburkholder** [comment]
LGTM. Running `plan` should be sufficient to assure this works with roles, right?

File: ci/pipeline.yml [hunk 1]
```diff
@@ -146,15 +146,15 @@ jobs:
     - get: plan-timer
       trigger: true
   - task: plan-update-tooling
+    tags:
```
Comment (jmcarp) [inline ci/pipeline.yml]: Do we need to tag the apply steps too?

Comment (rogeruiz) [inline ci/pipeline.yml]: ah yeah. thanks! 😆

Comment (rogeruiz) [inline ci/pipeline.yml]: @pburkholder let's pair on these changes that we need to do after standup

**Comment by cnelson** [comment]
I wouldn't nuke the stemcell user until we have resolution from AWS on the `ImportVolume` bug/issue.

**Comment by rogeruiz** [comment]
ah true. that was my mistake for mentioning that one to @pburkholder. _in a perfect world right?_ :trollface: I'll revert just those changes

**Pull request merged**